src/iop/colorequal.c

```diff
@@ -0,0 +1,1625 @@
+  dt_bauhaus_widget_set_label(g->sat_sliders[3], N_("saturation"), N_("green"));
+  dt_bauhaus_widget_set_label(g->sat_sliders[4], N_("saturation"), N_("turquoise"));
+  dt_bauhaus_widget_set_label(g->sat_sliders[5], N_("saturation"), N_("blue"));
+  dt_bauhaus_widget_set_label(g->sat_sliders[6], N_("saturation"), N_("lavender"));
+  dt_bauhaus_widget_set_label(g->sat_sliders[7], N_("saturation"), N_("purple"));
+
+  self->widget = dt_ui_notebook_page(g->notebook, N_("hue"), _("change hue hue-wise"));
+
+  g->smoothing_hue = dt_bauhaus_slider_from_params(self, "smoothing_hue");
+
+  g->hue_sliders[0] = g->hue_red = dt_bauhaus_slider_from_params(self, "hue_red");
+  g->hue_sliders[1] = g->hue_orange = dt_bauhaus_slider_from_params(self, "hue_orange");
+  g->hue_sliders[2] = g->hue_lime = dt_bauhaus_slider_from_params(self, "hue_lime");
+  g->hue_sliders[3] = g->hue_green = dt_bauhaus_slider_from_params(self, "hue_green");
+  g->hue_sliders[4] = g->hue_turquoise = dt_bauhaus_slider_from_params(self, "hue_turquoise");
+  g->hue_sliders[5] = g->hue_blue = dt_bauhaus_slider_from_params(self, "hue_blue");
+  g->hue_sliders[6] = g->hue_lavender = dt_bauhaus_slider_from_params(self, "hue_lavender");
+  g->hue_sliders[7] = g->hue_purple = dt_bauhaus_slider_from_params(self, "hue_purple");
+  dt_bauhaus_widget_set_label(g->hue_sliders[0], N_("hue"), N_("red"));
+  dt_bauhaus_widget_set_label(g->hue_sliders[1], N_("hue"), N_("orange"));
+  dt_bauhaus_widget_set_label(g->hue_sliders[2], N_("hue"), N_("lime"));
+  dt_bauhaus_widget_set_label(g->hue_sliders[3], N_("hue"), N_("green"));
+  dt_bauhaus_widget_set_label(g->hue_sliders[4], N_("hue"), N_("turquoise"));
+  dt_bauhaus_widget_set_label(g->hue_sliders[5], N_("hue"), N_("blue"));
+  dt_bauhaus_widget_set_label(g->hue_sliders[6], N_("hue"), N_("lavender"));
+  dt_bauhaus_widget_set_label(g->hue_sliders[7], N_("hue"), N_("purple"));
+
+  self->widget = dt_ui_notebook_page(g->notebook, N_("brightness"), _("change brightness hue-wise"));
+
+  g->smoothing_bright = dt_bauhaus_slider_from_params(self, "smoothing_brightness");
+
+  g->bright_sliders[0] = g->bright_red = dt_bauhaus_slider_from_params(self, "bright_red");
+  g->bright_sliders[1] = g->bright_orange = dt_bauhaus_slider_from_params(self, "bright_orange");
+  g->bright_sliders[2] = g->bright_lime = dt_bauhaus_slider_from_params(self, "bright_lime");
+  g->bright_sliders[3] = g->bright_green = dt_bauhaus_slider_from_params(self, "bright_green");
+  g->bright_sliders[4] = g->bright_turquoise = dt_bauhaus_slider_from_params(self, "bright_turquoise");
+  g->bright_sliders[5] = g->bright_blue = dt_bauhaus_slider_from_params(self, "bright_blue");
+  g->bright_sliders[6] = g->bright_lavender = dt_bauhaus_slider_from_params(self, "bright_lavender");
+  g->bright_sliders[7] = g->bright_purple = dt_bauhaus_slider_from_params(self, "bright_purple");
+  dt_bauhaus_widget_set_label(g->bright_sliders[0], N_("bright"), N_("red"));
```

Comment:
I would use `brightness` here instead of `bright` (name consistency with the notebook tab).